examples/workspace/WORKSPACE

```diff
@@ -0,0 +1,38 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "rules_cc",
+    sha256 = "d75a040c32954da0d308d3f2ea2ba735490f49b3a7aa3e4b40259ca4b814f825",
+    urls = ["https://github.com/bazelbuild/rules_cc/releases/download/0.0.10-rc1/rules_cc-0.0.10-rc1.tar.gz"],
+)
+
+# In real repo, this should be replaced with like this
```

Comment:
We could try uncommenting this and instead using `--override_repository` in the `.bazelrc` or CI flags to override the repo. That would make the example more copy-pastable.

Reply:
I think it would make sense once we have a release process for this repo plus a way to update the example automatically.

For now, I think `local_repository` makes the setup a bit easier while discouraging folks to copy-paste the example and use it as-is.
I will take 1 step further to change the commit hash and SHA256 to dummy value to signal that folks would need to modify these values after copy-pasting.